### PR TITLE
Chapter1/homework/10

### PR DESCRIPTION
**Pull request by inz-**

### CODE REVIEW

File: Src/BootCamp.Chapter/CaesarCipher.cs [hunk 1]
```diff
@@ -11,12 +11,30 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            if (message == null)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/CaesarCipher.cs]: or empty

File: Src/BootCamp.Chapter/CaesarCipher.cs [hunk 2]
```diff
@@ -11,12 +11,30 @@ public static class CaesarCipher
...
+            var byteArray = ASCIIEncoding.ASCII.GetBytes(message.ToCharArray());
+            for (int i = 0; i < message.Length; i++)
+            {
+                byteArray[i] += shift;
+            }
+
+            return new string(Encoding.ASCII.GetChars(byteArray));
         }
 
         public static string Decrypt(string message, byte shift)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/CaesarCipher.cs:17]: Please redo the decrypt method: what's the difference between encrypt and decrypt 😄 ?

**Review by Almantask — CHANGES_REQUESTED** [review]

**Comment by inz-** [comment]
Once again, missed the obvious.. Refactored

**Review by Almantask — APPROVED** [review]
Perfect :)

**Pull request closed without merging**